integration-tests/src/test/resources/config-test-parent/pom.xml

```diff
@@ -44,12 +44,9 @@
 
     <pluginRepositories>
         <pluginRepository>
-        <id>openmrs-repo</id>
-        <name>OpenMRS Nexus Repository</name>
-        <url>https://mavenrepo.openmrs.org/nexus/content/repositories/public</url>
```

Comment:
I think changing from `/public` to `/releases` here is unnecessary....

Reply:
Unfortunately I can't explain this [Bamboo build failure](https://ci.openmrs.org/download/CONTRIB-PMP-JOB1/build_logs/CONTRIB-PMP-JOB1-49.log). I had expected it to pass without this change.

Reply:
How is this change going to fix it? If the problem is that (for w/e reason) it's looking for the SNAPSHOT version, that's also not in the `/releases` repo (We have a `/snapshots` repo for those... It would be fine to _add_ the snapshots repo as another plugin repository, but there's no way this change will fix the problem you've cited.

Reply:
Ahh, I see. I had over looked the fact that `https://mavenrepo.openmrs.org/nexus/content/repositories/public` is not a SNAPSHOT repository.